app/models/EmailTemplate.scala

```diff
@@ -189,13 +193,17 @@ object EmailTemplate {
     val oneMonth = 1
 
     val dueDate = defermentStatementType match {
-      case "Excise" =>
+      case EXCISE_STT_TYPE =>
         Try {
           LocalDate.of(periodEndYear, periodEndMonth, EXCISE_DUTY_DUE_DATE)
         }.getOrElse {
           LocalDate.of(periodEndYear, periodEndMonth, EXCISE_DUTY_DUE_DATE_FEB)
         }
 
+      case DD1920_STT_TYPE => LocalDate.of(periodEndYear, periodEndMonth, DD1920_STT_DUE_DATE)
```

Comment:
Could you please add test cases for two new test cases for this method (createDutyDefermentDueDate)? If it takes time then leave it